packages/scheduler/cleanup.py

```
@@ -0,0 +1,114 @@
+            name = doc.get("name", "restored-job")
+            tags = doc.get("tags") or []
+            run_count = doc.get("run_count", 0)
+            
+            # Remove stale run-once jobs
+            if "run-once" in tags and run_count > 0:
+                try:
+                    remove_result = store.remove(job_id)
+                    if inspect.isawaitable(remove_result):
+                        await remove_result
+                except Exception:
+                    pass
```

Ruff (0.15.18)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)